### PR TITLE
EAP 7.3 Itests fix

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by robstryker** [comment]
is this ready for merge? or we want to testPR it a few more times?

**Comment by robstryker** [comment]
testPR

File: as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/reddeer/util/OperateServerTemplate.java [hunk 1]
```diff
@@ -144,7 +144,12 @@ public void startServer() {
 	}
 
 	public void restartServer() {
-		serversView.getServer(getServerName()).restart();
+		try {
+			serversView.getServer(getServerName()).restart();
+		} catch (WaitTimeoutExpiredException ex){
```
Comment (odockal) [inline as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/reddeer/util/OperateServerTemplate.java:149]: Are you sure with this? This seems to be very specific workaround...

Comment (odockal) [reply as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/reddeer/util/OperateServerTemplate.java:149]: We discussed this workaround with @jkopriva and this seems to be a workaround and result will be monitored.

**Pull request merged**